src/app/[locale]/(pages)/conferma-email/page.tsx

```diff
@@ -32,8 +33,12 @@ const EmailConfirmationPage = (): React.ReactElement => {
           .then(data => {
             setUrlParams({ token, email: data.profile_email });
           })
-          .catch(() => {
-            pushWithLocale(ROUTES.EMAIL_NOT_CONFIRMED);
+          .catch((error) => {
+            if(error.left?.filter((e: { value: string; }) => e.value === EmailValidationErrorStatusEnum.TOKEN_EXPIRED).length > 0) {
```

Comment:
What about moving this check

```ts
error.left?.filter((e: { value: string; }) => e.value === EmailValidationErrorStatusEnum.TOKEN_EXPIRED).length > 
```

inside an utility function for readability? What do you think?